ocs_ci/helpers/dr_helpers_ui.py

```
@@ -1838,3 +1838,879 @@ def navigate_using_fleet_virtualization(acm_obj):
+    if appset_workloads:
+        # Derive name from workload namespace (remove 'appset-' prefix if present)
+        for workload in appset_workloads:
+            # The ApplicationSet name is typically the namespace without 'appset-' prefix
+            # e.g., namespace 'appset-busybox-1-cephfs' -> app name 'busybox-1-cephfs'
+            namespace = workload.workload_namespace
+            if namespace.startswith("appset-"):
+                app_name = namespace[len("appset-") :]
+            else:
+                app_name = namespace
+            apps_to_verify.append(("AppSet/Managed", app_name))
+            log.info(f"Will verify AppSet workload (derived from namespace): {app_name}")
```

Ruff (0.16.1)

[warning] Logging statement uses f-string

(G004)